### PR TITLE
OSDOCS-4494(cont): Restructured the x1 and x2 instance types to a separate list

### PR DESCRIPTION
**Pull request by EricPonvelle**
Version(s):
Enterprise-4.11+

Issue:
[OSDOCS-4494](https://issues.redhat.com/browse/OSDOCS-4494) continuation

Link to docs preview:
- [OSD](https://53046--docspreview.netlify.app/openshift-dedicated/latest/osd_architecture/osd_policy/osd-service-definition.html#aws-instance-types-ccs_osd-service-definition)
- [ROSA](https://53046--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-service-definition.html#rosa-sdpolicy-aws-instance-types_rosa-service-definition)

Additional information:
Created a new list for the `x1` and `x2` instance types in ROSA and OSD.

### CODE REVIEW

**Review by arendej — APPROVED** [review]
LGTM

**Review by kelbrown20 — COMMENTED** [review]
Other than what I just added this LGTM!

File: modules/rosa-sdpolicy-am-aws-compute-types.adoc [hunk 1]
```diff
@@ -85,6 +85,37 @@
 - t3a.2xlarge (8 vCPU, 32 GiB)
 ====
 
+.Memory-intensive
+[%collapsible]
+====
+- x1.16xlarge (64 vCPU, 976GiB)
```
Comment (kelbrown20) [inline modules/rosa-sdpolicy-am-aws-compute-types.adoc:91]: In the other tables, the number and GiB has a space in between them. Like the one on line 116

**Review by kelbrown20 — COMMENTED** [review]
I just saw one thing after I submitted the review!

**Pull request merged**